menuconfig.py

```diff
@@ -3128,9 +3128,10 @@ def _getch_compat(win):
     # Uses get_wch() if available (Python 3.3+) and getch() otherwise. Also
     # handles a PDCurses resizing quirk.
 
-    if hasattr(win, "get_wch"):
+    try:
         c = win.get_wch()
-    else:
+        # curses.error is thrown in Python 3 on MacOs
```

Comment:
Could change the message to say

```
# Work around a crash where get_wch() raises curses.error on macOS when
# resizing the terminal by falling back on getch(). This needs more
# investigation. It doesn't happen on all systems.
```

See https://github.com/ulfalizer/Kconfiglib/issues/84#issuecomment-572450946 too. Would be nice to narrow down the configurations that have this issue.